src/seedsigner/gui/screens/seed_screens.py

```diff
@@ -416,18 +416,24 @@ def _run(self):
 @dataclass
 class SeedFinalizeScreen(ButtonListScreen):
     fingerprint: str = None
+    is_bip85_child_seed: bool = False
     is_bottom_list: bool = True
     button_data: list = None
 
     def __post_init__(self):
         self.show_back_button = False
-        self.title = _("Finalize Seed")
+        if self.is_bip85_child_seed:
+            self.title = _("Finalize Child Seed")
+            icon_color=GUIConstants.DIRE_WARNING_COLOR
+        else:
+            self.title = _("Finalize Seed")
+            icon_color=GUIConstants.INFO_COLOR
         super().__post_init__()
 
         self.fingerprint_icontl = IconTextLine(
             icon_name=SeedSignerIconConstants.FINGERPRINT,
-            icon_color=GUIConstants.INFO_COLOR,
-            icon_size=GUIConstants.ICON_FONT_SIZE + 12,
+            icon_color=icon_color,
+            icon_size=GUIConstants.ICON_FONT_SIZE * 1.5,
```

Comment:
I'm not sure how PIL handles non-integer sizes or if it matters at all. But we always cast to `int` just to be safe.